CONFIG_vita/CONFIG_vita.cmake

```diff
@@ -0,0 +1,45 @@
+# config app
```

Comment:
maybe CMAKE/platforms/vita.cmake

Reply:
I don't really like having a new top level folder just for the vita config app.

Let's relocate the Qt app into `CONFIG/qt` and put the vita config into `CONFIG/vita`